core_backend/app/dashboard/routers.py

```diff
@@ -25,9 +27,10 @@
 async def retrieve_overview_day(
     user_db: Annotated[UserDB, Depends(get_current_user)],
     asession: AsyncSession = Depends(get_async_session),
+    exclude_archived: bool = True,
```

Comment:
Should this be `False` by default?

Reply:
Same for the other endpoints

Reply:
Per our discussion, `exclude_archived` is actually not necessary here. Will remove all such parameters.